core/jobs/src/test/java/com/cognifide/aet/job/common/datafilters/statuscodesfilter/ExcludeStatusCodesFilterTest.java

```diff
@@ -34,20 +34,11 @@
 @RunWith(MockitoJUnitRunner.class)
 public class ExcludeStatusCodesFilterTest extends StatusCodesFilterTestBase {
 
-  @Test
```

Comment:
Why so many unit tests were removed? Shouldn't we replace them if logic changed?